### PR TITLE
Add "-j" option to specify number of jobs used for building

### PR DESCRIPTION
**Pull request by SGSSGene**
This option passes the "-j" option to make.

### CODE REVIEW

File: cmp_benchmarks.sh [hunk 1]
```diff
@@ -17,9 +17,12 @@ usage="Usage: ./cmp_bnchm.sh -m mode -n name_tag -b baseline -c contender \
     \n\
     Optional arguments:\n\
     -f : a threshold [0, 1] for filtering significant results, e.g. 0.2 . Defaults to 0.05 (5% difference).\n\
-    -o : full/path/to/save/results . Defaults to current directory.\n"
+    -o : full/path/to/save/results . Defaults to current directory.\n\
+    -j : number of jobs. This number is passed to make. Defaults to -1.\n"
```
Comment (wvdtoorn) [inline cmp_benchmarks.sh]: `-j : number of jobs. This number is passed to make. Defaults to 1.\n"`

(Does not match the default value below)

File: cmp_benchmarks.sh [hunk 2]
```diff
@@ -53,6 +57,13 @@ if ! [[ "$mode" =~ ^(results|execs|build)$ ]]; then
     exit 1
 fi
 
+if [[ "$jobct" -lt 1 ]]; then
+    echo -e "ERROR! number of jobs must be greater than 1.\n";
```
Comment (wvdtoorn) [inline cmp_benchmarks.sh]: `echo -e "ERROR! number of jobs must be at least 1.\n"`

**Review by wvdtoorn — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Just two minor things on the documentation.

File: cmp_benchmarks.sh (hunk 1 above)
Comment (SGSSGene) [reply cmp_benchmarks.sh:21]: Yes, I notice. I fixed this in one of the other PRs. Hope that works for you

**Review by wvdtoorn — APPROVED** [review]

**Pull request merged**